src/world_interface/real_world_constants.h

```diff
@@ -79,8 +79,10 @@ constexpr auto potMotors = frozen::make_unordered_map<motorid_t, potparams_t>({
 	 {.adc_lo = 123, .mdeg_lo = -200 * 1000, .adc_hi = 456, .mdeg_hi = 200 * 1000}},
 	{motorid_t::forearm,
 	 {.adc_lo = 1208, .mdeg_lo = -180 * 1000, .adc_hi = 841, .mdeg_hi = 180 * 1000}},
-	{motorid_t::wrist,
-	 {.adc_lo = 123, .mdeg_lo = -100 * 1000, .adc_hi = 456, .mdeg_hi = 100 * 1000}},
+	{motorid_t::wristDiffLeft,
+	 {.adc_lo = 0, .mdeg_lo = -100 * 0, .adc_hi = 0, .mdeg_hi = 100 * 0}},
+	{motorid_t::wristDiffRight,
+	 {.adc_lo = 0, .mdeg_lo = -100 * 0, .adc_hi = 0, .mdeg_hi = 100 * 0}},
```

Comment:
Are these values placeholders?  If they are, you should leave a comment so you remember to add the real values.

Reply:
Yes there should be a comment there I must've forgotten when I moved everything to the updated master